app/services/rooms/create/manager_create_action.rb

```diff
@@ -15,6 +15,7 @@ def initialize params
         def perform
           run_callbacks :perform do
             @room = SessionVote.new(params.except(:self_assign, :users_ids))
+            @room.status = :finish
```

Comment:
The return is a raise exception if save fails. You can use the `create` method. Replacing those 3 lines by 1:

`SessionVote.create(params.merge(status: :finish).except(:self_assign, :users_ids))`